### DISCUSSION TITLE
cersedb and horaedb

### DISCUSSION THREAD

**Discussion by tematou** [CeresDB/ceresdb#1351] [General]
so, What’s the difference between cersedb  and horaedb. Why does JD Cloud appear when searching for horaedb?
![screenshot-20231207-114133](https://github.com/CeresDB/horaedb/assets/35472988/411515d7-75d7-4eaa-9e95-602b5ea8806d)

https://zhuanlan.zhihu.com/p/253131245

please tell me why?

**Comment by tisonkun** [comment]
This is answered at https://github.com/CeresDB/horaedb/issues/1319#issuecomment-1846915287.
